_overviews/scala3-book/collections-methods.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Collections Methods
+type: section
+description: This page demonstrates the common methods on the Scala 3 collections classes.
+num: 38
+previous-page: collections-classes
+next-page: collections-summary
+---
+
+
+
+A great strength of Scala’s collections classes is that they come with dozens of pre-built methods, and those methods are consistently available across the immutable and mutable classes. The benefits of this are that you no longer need to write custom `for` loops every time you need to work on a collection, and when you move from one project to another, you’ll find these same methods used, rather than more custom `for` loops.
+
+There are *dozens* of methods available to you, so they aren’t all shown here. Instead, only some of the most commonly-used methods will be shown, including:
+
+- `map`
+- `filter`
+- `foreach`
+- `head`
+- `tail`
+- `take`, `takeWhile`
+- `drop`, `dropWhile`
+- `reduce`
+
+The following methods work on all of the sequence classes, including `List`, `Vector`, `ArrayBuffer`, etc., but these examples use a `List` unless otherwise specified.
```

Comment:
```suggestion
The following methods work on all of the sequence types, including `List`, `Vector`, `ArrayBuffer`, etc., but these examples use a `List` unless otherwise specified.
```